### ISSUE TITLE
Migrate to MIT license

### ISSUE THREAD

**Issue by ebekker** [PowerShellOrg/tug#91]
As per #90, we're going to migrate over to the MIT license for the Tug code base.  This keeps licensing consistent with other DSC tooling and resources.

Since we're migrating from GPL 3.0 to MIT, the *conditions of usage* are *less* restrictive so this should not be a problem for any users or forks.

However, and by the same token, the *agreement for contribution* is also *less* restrictive, so existing contributors need to agree to the transition with regard to releasing their past contributions under the new license.

To that end, we need to get an approval from contributors to move forward (this is one area where having a small contributor base comes in handy).  For everyone on this list, please add a comment on this ticket with the says the following:  **`I agree with a move to the MIT license for my contributions to this repository.`**

* @ebekker
* @concentrateddon 
* @majst32

**Comment by ebekker** [comment]
I agree with a move to the MIT license for my contributions to this repository.

**Comment by concentrateddon** [comment]
I agree with a move to the MIT license for my contributions to this repository.

**Comment by gaelcolas** [comment]
Pinging @majst32 here so we can get this one done ;)
